README.md

```diff
@@ -1,6 +1,11 @@
 # GovWifi Authentication API
 
-Authentication API for GovWifi
+This is the user authentication API for GovWifi.
```

Comment:
I don't think this adds much to the title without a description - think the best READMEs tell you what the thing _does_, not what it is.

Annoyingly, I haven't found a good description for this one yet (we're talking about it right now!), but think the right approach is just to lift these descriptions from [what I've written in the build repo](https://github.com/alphagov/govwifi-build/pull/282/files#diff-04c6e90faac2675aa89e2176d2eec7d8R13).

(Of course, I would think that, I wrote them :sweat_smile:)